tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -431,72 +431,132 @@ module.exports = {
         this.elements.fileLinkInFileRow.selector
     },
     /**
+     * function checks for the element whether listed or not on the filesList
+     * @param {string} element Name of the file/folder/resource
      *
-     * @param {string} element
+     * @returns {boolean}
      */
-    assertElementNotListed: function (element) {
-      return this
+    isElementListed: async function (element) {
+      let isListed = true
+      await this
         .waitForElementVisible('@filesTable')
         .useXpath()
         .waitForElementNotPresent('@loadingIndicator')
-        .waitForElementNotPresent(this.getFileRowSelectorByFileName(element))
+      await this
+        .waitForElementVisible({
```

Comment:
Why not use `this.elements` here. That will be more clear IMO

Reply:
It would be great!
:rocket: